data/keywords/tei/k000110.xml

```diff
@@ -78,12 +79,13 @@
    <text>
       <body>
          <entryFree xml:id="k000110" type="architectural-feature">
-            <term xml:id="term-k000110-1" xml:lang="en" type="preferred" source="#biblk000110-1">barrel tile ends (preferred)</term>
+            <term xml:id="term-k000110-8" xml:lang="zh-latn-pinyin-nt" type="preferred">wadang</term>
```

Comment:
Xuliang,
from GitHub, it looks like you have deleted source="#biblk000110-1" for barrel tile ends? 
it was 
<term xml:id="term-k000110-1" xml:lang="en" type="preferred" source="#biblk000110-1">barrel tile ends (preferred)</term>
now it looks like
 <term xml:id="term-k000110-1" xml:lang="en" source="#biblk000110-1">barrel tile ends</term>
source="#biblk000110-1" was removed?

if that is the case, would you put it back?
those are sources references that shouldn't be removed.